tailwind.config.js

```diff
@@ -102,6 +104,7 @@ module.exports = {
         sm: '28.125rem',
         md: '37.5rem',
         body: '41rem',
+        xlg: '44rem',
```

Comment:
Any ideas about another names? `x` is `extra`, like bigger `lg`?

Reply:
Renamed to `tablet`